installing/install_config/customizations.adoc

```diff
@@ -29,46 +29,55 @@ Custom Resource Definitions (CRDs) that are deployed on your cluster.
 |Configure the Kubernetes API server.
 
 |Authentication
-|You can configure the identity provider for your cluster.
+|You can configure the
+xref:../../authentication/understanding-identity-provider.adoc#understanding-identity-provider[identity provider]
+for your cluster.
 
 |Build
 |Configure build settings.
 
 |ClusterVersion
-|Configure the desired version of the cluster and how updates are handled.
+|In {product-title} {product-version}, you must not customize the ClusterVersion
+resource for production clusters. Instead, follow the process to
+xref:../../updating/updating-cluster.adoc#updating-cluster[update a cluster].
 
 |Console
 |Configure the web console.
 
 |DNS
-|Configure DNS.
+|To configure DNS by using the Network Operator, you must
+xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#installing-aws-network-customizations[customize networking during installation].
 
-|Feature
-|Configure feature gates.
+|FeatureGate
+|You can
+xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[enable a FeatureGate]
+so that you can use Tech Preview features.
 
 |Image
-|Configure image registries.
+|Configure the
+xref:../../openshift_images/configuring-samples-operator.adoc#configuring-samples-operator[samples Operator]
+to start using images.
```

Comment:
The samples operator and image configuration are related but not the same.  This belongs in a section called "Samples" not "Image".